app/views/albums/edit.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="albums__topNav">
+  <%= button_to 'Home', root_path, method: 'get', class: "albums__homeButton" %>
+</div>
+
+<% if @album.errors.messages.present? %>
+  <%= render partial: "errors", locals: { record: @album } %>
+<% end %>
+
+<%= form_for @album do |f| %>
+  <table class="albums__table">
+    <col style="width:50%">
+    <col style="width:50%">
```

Comment:
Tables are intended for data, not for layout. If you need to recreate a table-like effect, check out the `display: table` and `display: table-cell` CSS properties.